readme.txt

```diff
@@ -9,4 +9,5 @@ This project is aimed at showcasing a list of the cutest pet breeds that are ado
 1. **Breed Profiles**: Detailed profiles of various pet breeds considered the cutest
 2. **High Quality Images**: Each breed will be accompanied by its high qulaity image that shows its cuteness
 3. **Responsiveness**: A fully responsive design that can be accessed across all devices 
-4. **Login/Logout Functionality**: Users can login/logout and save their favourite pets 
\ No newline at end of file
+4. **Login/Logout Functionality**: Users can login/logout and save their favourite pets 
+5. **Social Sharing**: Options to share pet related contents on social media
```

Comment:
Consider adding a newline at the end of the file. This is a common convention that helps prevent any concatenation issues that might occur when multiple files are combined. [medium]